src/Glpi/Agent/Communication/AbstractRequest.php

```diff
@@ -634,7 +634,7 @@ public function handleContentType($type)
             case 'application/x-br':
             case 'application/x-compress-br':
                 if (!function_exists('brotli_compress')) {
-                    $this->addError('Brotli PHP extension is missing!', 415);
+                    throw new \RuntimeException("Brotli PHP extension is missing!");
```

Comment:
To reproduce the previous behaviour, you should do something like this:

```suggestion
                    $exception = new \Glpi\Exception\Http\HttpException(415, 'Brotli PHP extension is missing!');
                    $exception->setMessageToDisplay('Unsupported compression');
                    throw $exception;
```